mapper/dm_agent/src/batcher.rs

```diff
@@ -1,14 +1,14 @@
 use clock::{Clock, Timestamp};
 use mqtt_client::{Message, MqttClient, MqttMessageStream, Topic, TopicFilter};
-use std::{sync::Arc, time::Duration};
+use std::sync::Arc;
 use thin_edge_json::{
     group::MeasurementGrouper, measurement::FlatMeasurementVisitor,
     serialize::ThinEdgeJsonSerializer,
 };
 use tokio::{
     select,
     sync::mpsc::{UnboundedReceiver, UnboundedSender},
-    time::sleep,
+    time::{self, Duration},
```

Comment:
Is that `time::self` really needed? Looks strange for an external crate.

Reply:
Agree with Albin. I would prefer two lines here `use time` and `use time::Duration`.